### PR TITLE
add snap.svg.zpd v0.0.11

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/huei90/snap.svg.zpd
**Watch 15
  Star 67
 Fork 27**
@Amomo could u help me check this PR for #7007 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please remove [snap.svg](https://cdnjs.com/libraries/snap.svg), we have it and it doesn't belongs to snap.svg.zpd.

**Comment by maruilian11** [comment]
@Amomo i have removed `snap.svg.js`, could you help me check this PR? thank you!

File: ajax/libs/snap.svg.zpd/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ],
+  "author": "Huei Tan",
+  "license": "BSD-2-Clause",
+  "homepage": "https://github.com/huei90/snap.svg.zpd",
+  "npmName": "snap.svg.zpd",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "snap.svg.zpd.js"
```
Comment (PeterDaveHello) [inline ajax/libs/snap.svg.zpd/package.json]: `"snap.svg.zpd*.js"`?

**Comment by maruilian11** [comment]
@Amomo i have updateed npm auto-update config files,  could you help me check this PR? thank you!

File: ajax/libs/snap.svg.zpd/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "pan",
+    "drag",
+    "rotate",
+    "snap.svg",
+    "snap.svg plugin",
+    "plugin"
+  ],
+  "author": "Huei Tan",
+  "license": "BSD-2-Clause",
+  "homepage": "https://github.com/huei90/snap.svg.zpd",
```
Comment (PeterDaveHello) [inline ajax/libs/snap.svg.zpd/package.json]: https://huei90.github.io/snap.svg.zpd/

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Pull request merged**